### ISSUE TITLE
Annotations will not stay off

### ISSUE THREAD

**Issue by illBelief** [YePpHa/YouTubeCenter#2412]
Regardless of whether or not the the box is checked in player settings, annotations will not stay off when videos are played in new tab. Annotations were also turned off in youtube account settings.

**Comment by omperus** [comment]
Yeah, its already reported, the developer abandoned the project, so until someone makes a working fork, we are fucked.

**Comment by illBelief** [comment]
https://github.com/YePpHa/YouTubeCenter/issues/2287

This is working for me so far

**Comment by iivs** [comment]
Hide annotations no longer works on FF 53.0
Also remove watermark is broken and prevent autoplay.

**Comment by illBelief** [comment]
It's working for me on FF 53.0 64bit

What I did was install tampermonkey then copy paste all the code from the current version of youtube center into notepad++. Find all and replace all:

    fixHTML5Annotations: set it to true
    enableAnnotations: set to false
    channel_enableAnnotations: set to false
    embed_enableAnnotations: set to false

Copy your new code. Create a new script with tampermonkey and paste the code into there. Save it and run. Then disable the original youtube center.

**Comment by NewsGuyTor** [comment]
Just adding this to uBlock is a simple option:
`youtube.com###sb-button-notify
youtube.com###page > .alerts-wrapper
youtube.com###ticker
youtube.com##.iv-click-target.branding-img
||youtube.com/annotations_invideo?$object-subrequest
youtube.com##.video-annotations
youtube.com##.home .branded-page-v2-secondary-col
youtube.com##.feed .branded-page-v2-secondary-col
youtube.com##.guide-builder-v2 .branded-page-v2-secondary-col
||youtube.com/share_ajax?action_get_share_box=1&video_id=$xmlhttprequest
youtube.com##.action-panel-trigger[role="button"][data-trigger-for="action-panel-share"]
youtube.com##.yt-uix-button[data-trigger-for="action-panel-share"]
||ytimg.com/yts/swfbin/player-*/endscreen.swf$object-subrequest,domain=youtube.com
youtube.com##.html5-endscreen-content
youtube.com##.html5-endscreen
youtube.com###channel-subscription-promo-in-feed
youtube.com##.ytp-cards-button.ytp-button
youtube.com##.iv-promo-contents
youtube.com##.iv-click-target.iv-promo-img
youtube.com##.ytp-cards-teaser`
